CMakeLists.txt

```diff
@@ -1786,9 +1790,11 @@ elseif(WINDOWS)
   if(MINGW OR CYGWIN)
     list(APPEND EXTRA_LIBS mingw32)
     list(APPEND EXTRA_LDFLAGS "-mwindows")
-    set(SDL_CFLAGS "${SDL_CFLAGS} -Dmain=SDL_main")
-    list(APPEND SDL_LIBS "-lmingw32" "-lSDL2main" "-mwindows")
-  endif()
+    if(NOT SDL2_DISABLE_SDL2MAIN)
+      set(SDL_CFLAGS "${SDL_CFLAGS} -Dmain=SDL_main")
+      list(APPEND SDL_LIBS "-lmingw32" "-lSDL2main" "-mwindows")
+    endif(NOT SDL2_DISABLE_SDL2MAIN)
+endif()
```

Comment:
indent change (removal) here seems wrong

Reply:
You lost me. Is the indenting wrong, or the logic wrong?
The indenting looks fine to me...

Reply:
2 spaces were removed at the beginning of the endif() here.

Reply:
Oh, I see.
I also think I broke SDL_LIBS.
My latest push should handle both.